ImageD11/nbGui/S3DXRD/run_pbp_recon.py

```diff
@@ -0,0 +1,55 @@
+if __name__ == "__main__":
+    # horrible workaround to include id11 code path
+    import sys
+    
+    id11_code_path = sys.argv[1]
```

Comment:
Better to the environment variable PYTHONPATH outside of the script. e.g.
`PYTHONPATH={id11_code_path} {sys.executable} run_pbp_recon.py ...`

Reply:
Fixed in https://github.com/FABLE-3DXRD/ImageD11/pull/317/commits/6f3d38f9c70431ad1bc93b0fdb4dd44c93c48d1d